api/lib/domain/models/CompetenceAnswerCollectionForScoring.js

```diff
@@ -18,12 +18,14 @@ module.exports = class CompetenceAnswerCollection {
   numberOfCorrectAnswers() {
     let nbOfCorrectAnswers = 0;
     this.answers.forEach((answer) => {
-      if (this.answers.length < 3 && answer.isAFullyCorrectQROCMdep()) { // TODO : remove (useless) check on length ?
-        nbOfCorrectAnswers += 2;
-      } else if (this.answers.length < 3 && answer.isAPartiallyCorrectQROCMdep()) { // TODO : remove (useless) check on length ?
-        nbOfCorrectAnswers += 1;
-      } else if (answer.isCorrect()) {
-        nbOfCorrectAnswers += 1;
+      if (!answer.challenge.isNeutralized) {
+        if (this.answers.length < 3 && answer.isAFullyCorrectQROCMdep()) {
+          nbOfCorrectAnswers += 2;
+        } else if (this.answers.length < 3 && answer.isAPartiallyCorrectQROCMdep()) {
+          nbOfCorrectAnswers += 1;
+        } else if (answer.isCorrect()) {
+          nbOfCorrectAnswers += 1;
+        }
```

Comment:
Suggestion pour simplifier :
```suggestion
      if (answer.challenge.isNeutralized) {
        return;
      }
      
      if (this.answers.length < 3 && answer.isAFullyCorrectQROCMdep()) {
        nbOfCorrectAnswers += 2;
      } else if (this.answers.length < 3 && answer.isAPartiallyCorrectQROCMdep()) {
        nbOfCorrectAnswers += 1;
      } else if (answer.isCorrect()) {
        nbOfCorrectAnswers += 1;
      }
```